code/modules/antagonists/revenant/revenant.dm

```diff
@@ -376,11 +369,11 @@
 	message_admins("Revenant ectoplasm was left undestroyed for 1 minute and is reforming into a new revenant.")
 	forceMove(drop_location()) //In case it's in a backpack or someone's hand
 	revenant.forceMove(loc)
-	if(client_to_revive)
+	if(old_key)
 		for(var/mob/M in GLOB.dead_mob_list)
-			if(M.client == client_to_revive) //Only recreates the mob if the mob the client is in is dead
```

Comment:
client.key is just a string and won't get destroyed if the client disconnects, right? If so, this will fix #35390

Reply:
Yes, no idea why this even tried to use client.